backend/src/database/migrations/20230608034909-create-govsg-tables.js

```diff
@@ -0,0 +1,291 @@
+          model: 'govsg_templates',
+          key: 'id',
+        },
+        onUpdate: 'CASCADE',
+      },
+    })
+    await queryInterface.createTable('govsg_ops', {
+      id: {
+        autoIncrement: true,
+        type: Sequelize.DataTypes.BIGINT,
+        allowNull: false,
+        primaryKey: true,
+      },
+      campaign_id: {
+        type: Sequelize.DataTypes.BIGINT,
+        allowNull: true,
+        references: {
+          model: 'campaigns',
+          key: 'id',
+        },
+        onUpdate: 'CASCADE',
+      },
+      recipient: {
+        type: Sequelize.DataTypes.STRING(255),
+        allowNull: true,
+      },
+      params: {
+        type: Sequelize.DataTypes.JSONB,
+        allowNull: true,
+      },
+      service_provider_message_id: {
+        type: Sequelize.DataTypes.STRING(255),
+        allowNull: true,
+      },
+      error_code: {
+        type: Sequelize.DataTypes.STRING(255),
+        allowNull: true,
+      },
+      error_description: {
+        type: Sequelize.DataTypes.TEXT,
+        allowNull: true,
+      },
+      status: {
+        type: Sequelize.DataTypes.ENUM(Object.values(GovsgMessageStatus)),
+        allowNull: true,
+      },
+      accepted_at: {
+        type: Sequelize.DataTypes.DATE,
+        allowNull: true,
+      },
+      sent_at: {
+        type: Sequelize.DataTypes.DATE,
+        allowNull: true,
+      },
+      delivered_at: {
+        type: Sequelize.DataTypes.DATE,
+        allowNull: true,
+      },
+      read_at: {
+        type: Sequelize.DataTypes.DATE,
+        allowNull: true,
+      },
+      errored_at: {
+        type: Sequelize.DataTypes.DATE,
+        allowNull: true,
+      },
+      created_at: {
+        type: Sequelize.DataTypes.DATE,
+        allowNull: false,
+      },
+      updated_at: {
+        type: Sequelize.DataTypes.DATE,
+        allowNull: false,
+      },
+    })
+    await queryInterface.createTable('govsg_messages_transactional', {
+      id: {
+        autoIncrement: true,
+        type: Sequelize.DataTypes.BIGINT,
+        allowNull: false,
+        primaryKey: true,
+      },
+      template_id: {
+        type: Sequelize.DataTypes.BIGINT,
+        allowNull: true,
+        references: {
+          model: 'govsg_templates',
+          key: 'id',
+        },
+        onUpdate: 'CASCADE',
+      },
+      recipient: {
+        type: Sequelize.DataTypes.STRING(255),
+        allowNull: true,
+      },
+      params: {
+        type: Sequelize.DataTypes.JSONB,
+        allowNull: true,
+      },
+      service_provider_message_id: {
+        type: Sequelize.DataTypes.STRING(255),
+        allowNull: true,
+      },
+      error_code: {
+        type: Sequelize.DataTypes.STRING(255),
+        allowNull: true,
+      },
+      error_description: {
+        type: Sequelize.DataTypes.TEXT,
+        allowNull: true,
+      },
+      status: {
+        type: Sequelize.DataTypes.ENUM(Object.values(GovsgMessageStatus)),
+        allowNull: true,
```

Comment:
`allowNull` can be false right? since the enums should be exhaustive alr

Reply:
Can, but then I have to change the implementation for the db function a bit haha

Reply:
But will do it cos it makes sense